lib/openstudio-standards/standards/ashrae_90_1_prm/ashrae_90_1_prm.Model.rb

```diff
@@ -864,6 +864,115 @@ def model_get_fan_power_breakdown
     return true
   end
 
+  # Applies the HVAC parts of the template to all objects in the model using the the template specified in the model.
+  #
+  # @param model [OpenStudio::Model::Model] OpenStudio model object
+  # @param climate_zone [String] ASHRAE climate zone, e.g. 'ASHRAE 169-2013-4A'
+  # @param apply_controls [Bool] toggle whether to apply air loop and plant loop controls
+  # @param sql_db_vars_map [Hash] hash map
+  # @return [Bool] returns true if successful, false if not
+  def model_apply_hvac_efficiency_standard(model, climate_zone, apply_controls: true, sql_db_vars_map: nil)
+    sql_db_vars_map = {} if sql_db_vars_map.nil?
+
+    OpenStudio.logFree(OpenStudio::Info, 'openstudio.standards.Model', "Started applying HVAC efficiency standards for #{template} template.")
+
```

Comment:
This 90_1_prm version is missing the apply-controls and fan motor efficiency sections that are in the general version of the method. I'd like to split the controls sections to a separate method, since it is logically not about hvac efficiency values. Let's keep motor efficiency here, but assign it to a separate JIRA task, so we can get closure on the general HVAC efficiency task.

Reply:
Added the lines for fan and pump efficiency, but comment them. I add the TODO note. Will work on this in another task.